### PR TITLE
feat: blackjack

### PR DESCRIPTION
**Pull request by namgold**

### CODE REVIEW

File: src/components/swapv2/LimitOrder/ListOrder/useRequestCancelOrder.tsx [hunk 1]
```diff
@@ -30,15 +31,16 @@ import { CancelOrderFunction, CancelOrderResponse, CancelOrderType, LimitOrder }
 const useGetEncodeLimitOrder = () => {
   const { account } = useActiveWeb3React()
   const [getEncodeData] = useGetEncodeDataMutation()
-  const { library } = useWeb3React()
+  const { readProvider } = useKyberSwapConfig()
```
Comment (nguyenhoaidanh) [inline src/components/swapv2/LimitOrder/ListOrder/useRequestCancelOrder.tsx:34]: when using readProvider/library?

Comment (namgold) [reply src/components/swapv2/LimitOrder/ListOrder/useRequestCancelOrder.tsx:34]: - both of them is for interacting with rpc
- `readProvider` is for reading state of blockchain via rpc
- `library` is for signing transactions & send them to blockchain


- `readProvider` we use our custom rpc API
- `library` is readonly, return from web3-react/wallets e.g: metamask, rabby. We cannot customize the rpc API

For short:
- `readProvider` is for reading
- `library` is for writing

One more thing: from this PR, `library` will additional checking blackjack, `readProvider` not, letting user able to view the data. So we must use them correctly, if not, UI won't be able to show data correctly.

File: src/services/blackjack.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+type BlackjackResponse = {
+  data: {
+    wallets: BlackjackCheck[]
+  }
+}
+
+const blackjackApi = createApi({
+  reducerPath: 'blackjackApi',
+  baseQuery: baseQueryOauth({
```
Comment (nguyenhoaidanh) [inline src/services/blackjack.ts:20]: may get cors issue on prod.
If API don't need access token, use fetchBaseQuery instead.

Comment (namgold) [reply src/services/blackjack.ts:20]: thanks

File: src/constants/networks/görli.ts [hunk 3]
```diff
@@ -70,7 +70,7 @@ const görliInfo: EVMNetworkInfo = {
     staking: '0x9bc1214E28005e9c3f5E99Ff01C23D42796702CF',
     dao: '0x583c0A1a49CdC99f4709337fa5500844316366dc',
     rewardsDistributor: '0x62D82BC6aa44a4340F29E629b43859b7e0C1E915',
-    daoStatsApi: 'https://kyberswap-dao-stats.dev.kyberengineering.io',
+    daoStatsApi: 'https://kyberswap-dao-stats-testnet.dev.kyberengineering.io',
```
Comment (namgold) [inline src/constants/networks/görli.ts:73]: todo

File: .env.dev [hunk 4]
```diff
@@ -1,14 +1,14 @@
-VITE_AGGREGATOR_API=https://aggregator-api.dev.kyberengineering.io
-VITE_AGGREGATOR_STATS_API=https://aggregator-stats.dev.kyberengineering.io
+VITE_AGGREGATOR_API=https://aggregator-api.kyberswap.com
```
Comment (namgold) [inline .env.dev:1]: todo

File: .github/workflows/pr.yaml [hunk 5]
```diff
@@ -114,7 +114,7 @@ jobs:
           VITE_TAG: ${{ needs.prepare.outputs.image_tag }}
           CURRENT_BRANCH: ${{ needs.prepare.outputs.current_branch }}
           NODE_OPTIONS: '--max_old_space_size=4096'
-        run: yarn build
+        run: yarn build-dev
```
Comment (namgold) [inline .github/workflows/pr.yaml:117]: todo

**Review by viet-nv — APPROVED** [review]

**Review by viet-nv — APPROVED** [review]

**Pull request merged**